src/components/Utils/Validate.js

```diff
@@ -15,6 +15,15 @@ export const requiredRefNumber = (value, allValues) => {
     : undefined;
 };
 
+// Field is required only if 'vendor_detail.ref_number_type' isn't empty
+export const requiredRefNumberType = (value, allValues) => {
+  const refNumberType = allValues.vendor_detail.ref_number_type;
```

Comment:
Have you checked this on new Line? I think that `vendor_detail` might be absent and this line might cause an exception

Reply:
Checked. It works with new Line too.